### ISSUE TITLE
ILogger required

### ISSUE THREAD

**Issue by jasdefer** [jacqueskang/IpcServiceFramework#37]
Thank you very much for this repository!
The setup of am `ILogger` is required when using the `AddTcpEndpoint`. Otherwise, an exception is thrown. The quick start does not contain the injection of the `ILogger`.

```csharp
services
    .AddLogging(builder =>
    {
        builder.AddConsole();
        builder.SetMinimumLevel(LogLevel.Debug);
     });
```

**Comment by jacqueskang** [comment]
I'll fix this, thanks for reporting

**Comment by jacqueskang** [comment]
resolved in v2.0.2
